library/src/com/jeremyfeinstein/slidingmenu/lib/CustomViewAbove.java

```diff
@@ -17,18 +17,18 @@
 import android.view.animation.Interpolator;
 import android.widget.Scroller;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import com.jeremyfeinstein.slidingmenu.lib.SlidingMenu.OnClosedListener;
-import com.jeremyfeinstein.slidingmenu.lib.SlidingMenu.OnOpenedListener;
-
 import androidx.annotation.DrawableRes;
 import androidx.core.view.MotionEventCompat;
 import androidx.core.view.VelocityTrackerCompat;
 import androidx.core.view.ViewCompat;
 import androidx.core.view.ViewConfigurationCompat;
 
+import java.util.ArrayList;
```

Comment:
it seems like the change of imports was reverted, but only on a few places. was that intentional?